.github/workflows/pull-request-testing.yml

```diff
@@ -15,33 +15,8 @@ env:
   IMAGE_NAME: ${{ github.repository }}
 
 jobs:
-  dependencies:
```

Comment:
I didn't notice much difference in terms of time, in any case, the first job "dependencies" is not useful because do the same thing as the second "unit" (now "test").
I will put dependencies back anyway on the second and remove the first